server/views/pages/bookedtoday/arrivals/confirmArrival/checkAnswersForCreateNewRecord.njk

```diff
@@ -0,0 +1,72 @@
+{% extends "../../../../partials/layout.njk" %}
+{% set pageTitle = "Check answers" %}
+
+{% block content %}
+    <div class="govuk-width-container">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <span class="govuk-caption-xl"> Confirm a prisoner's arrival </span>
+                <h1 class="govuk-heading-l govuk-!-margin-bottom-7">You're about to add this person to the establishment roll</h1>
+                <dl class="govuk-summary-list">
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Name
+                        </dt>
+                        <dd class="govuk-summary-list__value" data-qa="prisoner-name">
+                            {{ data.firstName}} {{ data.lastName }}
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
```

Comment:
summary-list has not been extracted into a Nunjucks macro because the two pages on which it is used, have slightly different content